static/src/stylesheets/module/atoms/_storyquestions.scss

```diff
@@ -4,6 +4,7 @@
 
     &.submeta {
         border: 0;
+        padding: ($gs-baseline / 3) * 4;
```

Comment:
Just to check because previously I may have misunderstood, but how I read this before `($gs-baseline / 3)  4;` compiles to `4px 4px` where this compiles to `16px`

Which one is your intention?